docs/hoon/library/2k.md

```diff
@@ -180,164 +174,192 @@ Source
         ?:  |(?=($~ q.b) (vor n.a n.q.b))
           [n.a l.a q.b]
         [n.q.b [n.a l.a l.q.b] r.q.b]
-      ::
 
-Examples
---------
 
-    > =s (~(gas to *(qeu @)) `(list @)`~[1 2 3])
+#### Examples
+
+    > =s (~(gas to *(tree @)) `(list @)`~[1 2 3])
     > ~(get to s)
-    [p=1 q={3 2}]
+    [n=2 l={3} r={1}]
+
     > ~(get to ~)
     ! exit
 
-------------------------------------------------------------------------
-
+***
 ### `+-nap:to`
 
-Remove last element 
+Remove head
 
-Removes the head of `++queue` `a`, producing the resulting queue.
+Removes the head of queue `a`, producing the resulting queue.
 
-Accepts
--------
+#### Accepts
 
 `a` is a queue.
 
-Produces
---------
+#### Produces
 
 A queue.
 
-Source
-------
+#### Source
 
-      +-  nap                                               ::  removes head
+      +-  nap                                              
         ?>  ?=(^ a)
```

Comment:
Looks like this errors out on an empty queue. That should be documented.

Reply:
Interesting, will do so